test/scalability/mzcompose.py

```diff
@@ -38,20 +40,38 @@
 
 
 def execute_operation(
-    args: tuple[Workload, int, Operation, Cursor, int, int]
+    args: tuple[Workload, int, list[Operation], Cursor, int, int]
 ) -> dict[str, Any]:
-    workload, concurrency, operation, cursor, i1, i2 = args
-
-    start = time.time()
-    operation.execute(cursor)
-    wallclock = time.time() - start
-
-    return {
-        "concurrency": concurrency,
-        "wallclock": wallclock,
-        "operation": type(operation).__name__,
-        "workload": type(workload).__name__,
-    }
+    workload, concurrency, operations, cursor, i1, i2 = args
+
+    results = list()
+    for operation in operations:
+        start = time.time()
+        operation.execute(cursor)
+        wallclock = time.time() - start
+
+        results.append({
+            "concurrency": concurrency,
+            "wallclock": wallclock,
+            "operation": type(operation).__name__,
+            "workload": type(workload).__name__,
+        })
+
+    return results
+
+def open_cursor(endpoint: Endpoint, connect_sqls: list[str]):
+    conn = None
+    while conn == None:
```

Comment:
I believe this masks a bug of some sort in the product. So I have left it out of my replacement PR. How often does it happen to you?

Reply:
happened quite a bit when I was running this frequently on my local machine, but I can keep monitoring and come back to you once I use it some more again